src/otx/algo/detection/atss.py

```diff
@@ -33,11 +34,35 @@
     from otx.core.types.label import LabelInfoTypes
 
 
+AVAILABLE_MODEL_VERSIONS: list[str] = [
+    "atss_mobilenetv2",
+    "atss_resnext101",
```

Comment:
Do we need the word `atss_`? class already tells us that this is an ATSS.

Reply:
I think so, but if it is changed, others should be changed such as `yolox_tiny` -> `tiny` or `segnext_base` -> `base` like classification use case. Just like we used to use `variant` before.